python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -44,6 +44,13 @@
 def _replace_with_int4_linear(model, modules_to_not_convert=None,
                               current_key_name=None, convert_shape_only=False):
     has_been_replaced = False
+    model_state_dict = model.state_dict()
+    for name, param in model.named_parameters():
+        if param.data.device == torch.device('meta'):
+            from accelerate.utils.modeling import set_module_tensor_to_device
+            param = model_state_dict[name]
+            set_module_tensor_to_device(model, name, "cpu", torch.empty(*param.size(), dtype=torch.float32))
+   
```

Comment:
By our method, some layers inited in device("meta") (associated with `low_cpu_mem_usage`'s lazy init strategy), are no properly move back to cpu somehow. so we move these layer back to cpu here to avoid NoImplementnError. Refer to:

https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_utils.py#L3110

Reply:
@yangw1234 hi, would you mind taking a look at this?

Reply:
LGTM

Maybe also adding a few words of comments?

Reply:
sure